jvb/src/test/kotlin/org/jitsi/videobridge/ConferenceTest.kt

```diff
@@ -17,53 +17,33 @@ package org.jitsi.videobridge
 
 import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.matchers.shouldBe
-import io.mockk.every
-import io.mockk.mockk
-import io.mockk.mockkStatic
 import org.jitsi.ConfigTest
-import org.jitsi.videobridge.octo.OctoRelayService
-import org.jitsi.videobridge.octo.OctoRelayServiceProvider
-import org.jitsi.videobridge.octo.singleton
+import org.jitsi.videobridge.octo.singleton as octoRelayServiceProvider
 import org.json.simple.JSONObject
 import org.json.simple.parser.JSONParser
 import org.jxmpp.jid.impl.JidCreate
-import org.osgi.framework.BundleContext
 
 /**
  * This is a high-level test for [Conference] and related functionality.
  */
 class ConferenceTest : ConfigTest() {
-    private val octoRelayService = OctoRelayService()
-    private val octoRelaySupplier: OctoRelayServiceProvider = mockk() {
-        every { get() } returns octoRelayService
-    }
-
-    private val bundleContext = mockk<BundleContext> {
-        every { registerService(any() as String, any(), any()) } returns null
-    }
-    private val videobridge = mockk<Videobridge> {
-        every { bundleContext } returns this@ConferenceTest.bundleContext
-    }
-
-    init {
-        mockkStatic("org.jitsi.videobridge.octo.OctoRelayServiceProviderKt")
-        every { singleton() } returns octoRelaySupplier
-    }
+    private val videobridge = Videobridge()
 
     init {
         val name = JidCreate.entityBareFrom("roomName@somedomain.com")
-        context("Adding local endpoints should work") {
-            withNewConfig(newConfigOctoEnabled, loadDefaults = true) {
+
+        withNewConfig(newConfigOctoEnabled, loadDefaults = true) {
```

Comment:
Hmm, from what I've seen, using `withNewConfig` at this level will not work correctly because `context` merely registers a test, so the context block is actually run _after_ the `withNewConfig` block finishes, meaning it restores the old config.

Reply:
I don't understand how that even worked before. Fixed now